public/apps/configuration/utils/tenant-utils.tsx

```diff
@@ -179,6 +179,33 @@ export function transformRoleTenantPermissions(
   }));
 }
 
+export function getNamespacesToRegister(accountInfo: any) {
+  const tenants = accountInfo.tenants || {};
+  const availableTenantNames = Object.keys(tenants!);
+  const namespacesToRegister = availableTenantNames.map((tenant) => {
+    if (tenant === globalTenantName) {
+      return {
+        id: GLOBAL_USER_DICT.Value,
+        name: GLOBAL_USER_DICT.Label,
+      };
+    } else if (tenant === accountInfo.user_name) {
+      return {
+        id: `${PRIVATE_USER_DICT.Value}${accountInfo.user_name}`,
+        name: PRIVATE_USER_DICT.Label,
+      };
+    }
+    return {
+      id: tenant,
+      name: tenant,
+    };
+  });
+  namespacesToRegister.push({
```

Comment:
This will always include `default` in the dropdown. `default` comes into play when OSD deployments with existing saved objects enable the saved object aggregation view feature. These objects do not have the `namespace` of the saved object populated and will be given the `default` (or `null`) namespace.